### PR TITLE
Calculate file and package fitness

### PR DESCRIPTION
**Pull request by isoos**
As usual, naming is hard. `Fitness` is a lightweight version of `health`, with the major difference that `health` would get a deeper analysis, for example:
- running code (e.g. tests)
- code coverage
- readability

It is proportional to the library size (see https://github.com/dart-lang/pub-dartlang-dart/issues/305).

### CODE REVIEW

File: lib/src/fitness.dart [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+import 'analyzer_output.dart';
+import 'platform.dart';
+import 'summary.dart';
+
+part 'fitness.g.dart';
+
+@JsonSerializable()
+class Fitness extends Object with _$FitnessSerializerMixin {
+  final double value;
```
Comment (kevmoo) [inline lib/src/fitness.dart]: doc comments on these would be nice – what's the difference between them?

**Comment by kevmoo** [comment]
@isoos - will need to rebase w/ the removal of your last commit

**Comment by isoos** [comment]
Rebased/updated.

Just noticed: the complexity calculation have one weak spot: it doesn't traverse the part 'whatever.dart' statements (yet).

**Comment by isoos** [comment]
> Just noticed: the complexity calculation have one weak spot: it doesn't traverse the part 'whatever.dart' statements (yet).

Never mind, this is a non-issue: the parts are treated as a separate dependency anyway.

**Pull request merged**